app/src/main/java/org/mozilla/fenix/addons/WebExtensionActionPopupFragment.kt

```diff
@@ -44,7 +41,7 @@ class WebExtensionActionPopupFragment : Fragment(), EngineSession.Observer {
     ): View? {
         // Grab the [EngineSession] from the store when the view is created if it is available.
         if (engineSession == null) {
-            engineSession = coreComponents.store.state.extensions[args.webExtensionId]?.popupSession
+            createSession(coreComponents.store.state.extensions[args.webExtensionId]?.popupSession)
```

Comment:
What about when we consume the `popupSession` from the store below?

Reply:
What about it :) ?

Reply:
Shouldn't we make sure to dispatch `AddCustomTabAction` for that as well?

Reply:
@psymoon no, we don't need to do that, as this is handled by the base class in createView/destroyView.